package.json

```diff
@@ -47,12 +49,10 @@
   },
   "scripts": {
     "start": "gulp build && concurrently -r \"gulp watch\" webpack-dev-server",
-    "format": "prettier --print-width 100 --trailing-comma es5 --parser typescript --write --list-different app/**/*.ts app/**/*.tsx page/**/*ts webpack.config.js gulpfile.js",
-    "format:fix": "prettier --print-width 100 --trailing-comma es5 --parser typescript --write app/**/*.ts app/**/*.tsx page/**/*ts webpack.config.js gulpfile.js",
-    "tslint": "tslint -c tslint.json 'app/**/*.ts' 'app/**/*.tsx'",
-    "tslint:fix": "tslint --fix -c tslint.json 'app/**/*.ts' 'app/**/*.tsx'",
-    "lint": "yarn format && yarn tslint",
-    "lint:fix": "yarn format:fix && yarn tslint:fix",
+    "tslint": "tslint --fix 'app/*.ts?(x)' 'app/**/*.ts?(x)' 'page/ts/*.ts?(x)'",
```

Comment:
Maybe one of them should have the --fix taken off currently tslint and tslint:fix do the same thing